.github/CODE_OF_CONDUCT.md

```diff
@@ -1,22 +1,22 @@
-# Code of Conduct
+# Code of Conduct [![Coc](https://img.shields.io/badge/</>-CodeofConduct-<COLOR>.svg)](https://shields.io/)
 
-## 1. Purpose
+## 1. `Purpose`
```

Comment:
Is this commit from another issue? I don't this is needed for this PR.

Reply:
Oh sorry actually yes this was my previous issues i forgot to remove it 
So should close this PR and do it again removing it ?

Reply:
Although this approach would work. I would recommend that you try and use git commands to fix this. 

Also, keep in mind to separate the issues you are working on using branches, this helps in preventing unexpected commits popping up in across different issues.

Reply:
Yes i have created a separate branch for this issue k i don't how this happened but I'll keep this in mind and will do another PR thanks for helping